doto-frontend/src/components/pages/Calendar/CalendarComponent.js

```diff
@@ -111,6 +115,11 @@ export function Content({
 
     return (
         <AppointmentTooltip.Content {...restProps} appointmentData={appointmentData}>
+            <Grid container alignItems="center">
+                <div className={classes.dueContainer}>
+                    <div>Due: {new Date(appointmentData.dueDate).toLocaleString()}</div>
```

Comment:
Maybe word it as "Due By" ?
Just an opinion not a blocker

Reply:
Hmm I could add this in, makes it a little longer